docs/reference/contributing/guidelines/workflow.md

```diff
@@ -49,3 +49,47 @@ Pull requests on GitHub have to meet the following requirements in order to keep
 	1. Use the body to explain _what_ and _why_ vs _how_.
 * Because we use GitHub and explicit CLAs, special commit tags that other projects may use, such as “Reviewed-by”, or “Signed-off-by”, are redundant and should be omitted. GitHub keeps track of who reviewed what and when, and our stack of signed CLAs shows us who has agreed to our development contribution agreement.
 * Prefixing your commit message with a domain is acceptable and recommended where it makes sense to do so. However, prefixing one's domain with the name of the repo is not useful. For example, making a commit entitled "mbed-drivers: Fix doppelwidget frobulation" to the `mbed-drivers` repo would not be acceptable, as it is already understood that the commit applies to `mbed-drivers`. Renaming the commit to "doppelwidget: Fix frobulation" would be better, if we presume that "doppelwidget" is a meaningful domain for changes, as it communicates that the change applies to the doppelwidget area of mbed-drivers.
+
+#### GitHub pull requests workflow
+
+Each pull request will go through the following workflow:
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/workflow.jpg)<span>The workflow of merging a pull request</span></span>
+
+##### Pull request states
+
+Labels that the Mbed OS maintainers add to a pull request represent the pull request workflow states. The Mbed OS maintainers are responsible for moving pull requests through the workflow states.
```

Comment:
Could you please create a sentence or two explaining who the Mbed OS maintainers are and what they do at the start of this section?

Reply:
Shall we name them, and provide github names?

Reply:
Seems reasonable, the only reason I didn't specifically name them is in the future the list may change/grow etc and thus mean updating this documentation. By referring to the github group instead this covers them. But I am happy to do so.

Reply:
Your changes are fine, thanks @AnotherButler